cli/dcoscli/data/help/task.txt

```diff
@@ -34,6 +42,10 @@ Options:
         Print version information.
 
 Positional Arguments:
+    <cmd>
+        The command to execute inside the remote container, example: `/bin/bash`.
```

Comment:
Suggest: `The command to execute inside of the remote container. For example: `/bin/bash`.`